tests/workflow/testdata/deploy_docker_eden.txt

```diff
@@ -30,9 +30,7 @@ EDEN={{EdenConfig "eden.root"}}/{{EdenConfig "eden.bin-dist"}}/{{EdenConfig "ede
 
 $EDEN pod ps | grep nginx || exit
 
-URL=`$EDEN pod ps | grep nginx | awk '{print $4}'`
-
-test -n "$URL" && until curl $URL | grep 'Welcome to nginx'; do sleep 3; done
+until $EDEN sdn fwd eth0 8028 -- curl FWD_IP:FWD_PORT | grep 'Welcome to nginx'; do sleep 3; done
```

Comment:
Do we plan to use new notation in all tests where we want to access an app? If so, I think we should decide what to do with GCP tests. We [set](https://github.com/milan-zededa/eden/blob/35a1f570b65d5f16c6b3a600e983968c2e506bc9/pkg/models/gcp.go#L30) `eve.remote` internally for GCP devmodel and new implementation will [return](https://github.com/milan-zededa/eden/blob/35a1f570b65d5f16c6b3a600e983968c2e506bc9/cmd/sdn.go#L422) error.

Reply:
Fixed, now the function can handle any supported setup.